iib/workers/tasks/build_regenerate_bundle.py

```diff
@@ -118,7 +118,7 @@ def handle_regenerate_bundle_request(
                 _push_image(request_id, arch)
 
     set_request_state(request_id, 'in_progress', 'Creating the manifest list')
-    output_pull_spec = _create_and_push_manifest_list(request_id, arches)
+    output_pull_spec = _create_and_push_manifest_list(request_id, arches, [])
```

Comment:
You are accepting `build_tags` from the user in the API and not using them at all. I think we shouldn't give an option at all, no?
Am I missing something?

Reply:
it's shared method, so if you don't want to use them, you need to pass there empty list. Build_tags are not accepted for regenerate_bundles. Or do you think we should support it as well?

Reply:
No, I agree they shouldn't be supported in regenerate-bundle.